bee-message/Cargo.toml

```diff
@@ -41,6 +41,7 @@ bee-test = { path = "../bee-test", default-features = false }
 num_cpus = { version = "1.13.0", default-features = false }
 
 [features]
+chrysalis = [ ]
```

Comment:
A bit confusing to have `cpt2` and `chrysalis` features at the same time. They are mutually exclusive, right? We will enter muddy waters with this as compiling with `all-features` won't work anymore possibly breaking our CI (or is there an easy solution) If we go with this then I'ld suggest the term  `compat`/ `back-compat` or something else in this direction.

Reply:
Actually `chrysalis` feature is not that important as it only enable us to `un/pack` Message in chrysalis mode. 
we could easily handle that part in our chronicle, gonna remove it.